label_studio/io_storages/azure_blob/api.py

```diff
@@ -9,33 +9,105 @@
 
 
 class AzureBlobImportStorageListAPI(ImportStorageListAPI):
```

Comment:
Azure storage docs doesn't appear. The same problems with 
* GCS
* Redis
* Localfiles
![image](https://user-images.githubusercontent.com/501780/114946893-ba999180-9e54-11eb-8686-d7c5324fa849.png)

Reply:
are you sure using the latest commit in this branch? because all these titles have been fixed here